templates/k-bims-ledet/0/docker-compose.yml

```diff
@@ -170,6 +178,129 @@ services:
+      # we use geoserver host name provided by rancher internal dns resolver
+      GEOSERVER_LOCATION: http://geoserver:8080/geoserver/
+      UWSGI_CMD: uwsgi --ini /usr/src/ledet/uwsgi.conf
+      GEONODE_GEODATABASE: geonode_data
+      GEONODE_DATABASE_PASSWORD: docker
+      C_FORCE_ROOT: 1
+      BING_MAP_KEY: ${BING_MAP_KEY}
+      IUCN_API_KEY: ${IUCN_API_KEY}
+      MAP_TILER_KEY: ${MAP_TILER_KEY}
+      SENTRY_KEY: ${SENTRY_KEY}
+    entrypoint: /bin/sh
+    command: -c "/usr/local/bin/python manage.py runmessaging -i"
+    volumes:
+      - django-static:/home/web/static:rw
+      - django-media:/home/web/media:rw
+      - django-reports:/home/web/reports
+      - django-logs:/var/log/
+    links:
+      - geoserver:geoserver
+      - rabbitmq:rabbitmq
+      - db:db
+      - geonode-db
+    labels:
+      io.rancher.container.pull_image: always
+    restart: unless-stopped
+
+  worker:
+    image: kartoza/ledet_uwsgi:latest
+    environment:
+      DATABASE_HOST: db
+      DATABASE_NAME: gis
+      DATABASE_PASSWORD: docker
+      DATABASE_USERNAME: docker
+      DJANGO_SETTINGS_MODULE: ledet_core.settings.prod_docker
+      RABBITMQ_HOST: rabbitmq
+      VIRTUAL_HOST: ${SITE_URL}
+      VIRTUAL_PORT: '8000'
+      SITE_DOMAIN_NAME: http://${SITE_URL}
+      SITEURL: http://${SITE_URL}/
+      PYTHONPATH: /usr/src/ledet:/usr/src/app:/usr/src/bims:/home/web/django_project
+      HAYSTACK_HOST: elasticsearch
+      GEOCONTEXT_URL: ${GEOCONTEXT_URL}
+      GEOCONTEXT_COLLECTION_KEY: ${GEOCONTEXT_COLLECTION_KEY}
+      CONTACT_US_EMAIL: ${ADMIN_EMAIL}
+      ROOT_URLCONF: ledet_core.urls
+      # In this setting, we access GeoServer using Rancher Load Balancer, so
+      # it will be available from public internet
+      GEOSERVER_PUBLIC_LOCATION: http://${SITE_URL}/geoserver/
+      STATIC_ROOT: /home/web/static
+      GEONODE_DATABASE: gis
+      GEONODE_DATABASE_USER: docker
+      MEDIA_ROOT: /home/web/media
+      ADMIN_EMAILS: ${ADMIN_EMAIL}
+      GEOSERVER_ADMIN_PASSWORD: ${GEOSERVER_ADMIN_PASSWORD}
+      GEOIP_PATH: /home/web/media/geoip.db
+      GEONODE_GEODATABASE_PASSWORD: geonode_data
+      GEONODE_INSTANCE_NAME: geonode
+      DEFAULT_BACKEND_DATASTORE: datastore
+      IS_CELERY: 'False'
+      ALLOWED_HOSTS: '["localhost","127.0.0.1","${SITE_URL}"]'
+      # In this setting, GeoNode needs to access GeoServer from internal network
+      # we use geoserver host name provided by rancher internal dns resolver
+      GEOSERVER_LOCATION: http://geoserver:8080/geoserver/
+      UWSGI_CMD: uwsgi --ini /usr/src/ledet/uwsgi.conf
+      GEONODE_GEODATABASE: geonode_data
+      GEONODE_DATABASE_PASSWORD: docker
+      C_FORCE_ROOT: 1
+      BING_MAP_KEY: ${BING_MAP_KEY}
+      IUCN_API_KEY: ${IUCN_API_KEY}
+      MAP_TILER_KEY: ${MAP_TILER_KEY}
+      SENTRY_KEY: ${SENTRY_KEY}
+    entrypoint: /bin/sh
+    command: -c "celery worker --app=bims.celery:app -B  -l DEBUG -Q default,geonode,cleanup,email,update"
```

Comment:
We limit the queue to GeoNode basic queue.
Should also include BIMS queue if there's any.